package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aragon/os",
-  "version": "5.0.0-beta.4",
+  "version": "5.0.0-beta.6",
```

Comment:
You should probably update this branch with the latest `next` changes

Reply:
Done